tests/src/org.eclipse.jetty/jetty-client/11.0.12/src/test/java/org_eclipse_jetty/jetty_client/JettyClientTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package org_eclipse_jetty.jetty_client;
+
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.client.api.ContentResponse;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class JettyClientTest {
+    @Test
+    void test() throws Exception {
+        HttpClient client = new HttpClient();
+        client.start();
+        try {
+            ContentResponse response = client.GET("https://httpbin.org/get");
```

Comment:
What does this URL represents? Why it is used? Can you implement this test using localhost?

Reply:
Sure, I can. But then I think to have to use docker. What's the problem with non-localhost?

Reply:
No, its fine, understand that this is only testing a GET request.